data/updater.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python2
+
```

Comment:
And/or, given the place where this script looks for configuration files, it would perhaps be better to just give it as name `lepidopter-update.py`